meson.build

```diff
@@ -0,0 +1,26 @@
+project(
+  'starfit',
+  'fortran',
+  'c',
+# Run command to rewrite version number:
+# meson rewrite kwargs set project / version $(git describe --tags)
+version : '0.0.0',
```

Comment:
It's quite possible to set this dynamically, exactly the same way you do with setuptools_scm.

```meson
version: run_command('python3', '-m', 'setuptools_scm', check: true).stdout().strip()
```

Reply:
Thank you for all the pointers! We're just starting to learn about Meson, so this is very helpful.